dropwizard-index-page/src/main/java/com/palantir/indexpage/ClasspathAsset.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.indexpage;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.github.mustachejava.DefaultMustacheFactory;
+import com.github.mustachejava.Mustache;
+import com.github.mustachejava.MustacheFactory;
+import com.google.common.base.Charsets;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+/**
+ * Used to represent assets loaded from the classpath.
+ */
+public final class ClasspathAsset implements Asset {
+
+    private final Mustache mustacheTemplate;
+
+    public ClasspathAsset(URL resource) {
+        checkNotNull(resource);
+
+        MustacheFactory factory = new DefaultMustacheFactory();
+        BufferedReader reader;
+        try {
+            reader = new BufferedReader(new InputStreamReader(resource.openStream(), Charsets.UTF_8));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
talked offline, this should fail at startup because the file should exist.

can you throw a better error? it's unclear what to do if this gets thrown